bifrost/proxy/pages/passthru/onRenderClient.tsx

```diff
@@ -0,0 +1,4 @@
+export default async function onRenderClient() {
+  // Passthru pages are unhandled by bifrost, reload to grab legacy content from server
+  window.location.reload();
```

Comment:
this behavior always happened but used to be implicit. vike would see navigation to a page it couldnt handle and reload